Adding Symbol Map File

jpc-ae opened the pull request:
SVG sprites are unnecessary, since the use tag works on any kind of file. The only advantage a sprite file has is that it is viewable directly. This can be added to a map file if needed.

jpc-ae commented:
I signed it!

shyndman commented:
Hey @jpc-ae,

Looks awesome! Thanks for bringing this technique to our attention.

I'm going to script this up for repeatability. In the meantime, could you add a note to the root README.md similar to what we have for sprite sheets?

shyndman commented on iconmaps/README.md:
````diff
@@ -0,0 +1,42 @@
+To add an icon, place the following in your document wherever it is needed:
+
+```HTML
+<svg class="svg-ic_24px">
````
I'd prefer to stick to dashes only in class names.

jpc-ae commented:
I've updated my branch with several changes and the documentation regarding use. I updated the index.html file as well and prettified it as best I could.

Regarding class names, I was following the conventions used throughout the project. I personally prefer to stick with dashes as well.

jpc-ae commented:
I'm actually using a gulpfule to generate my files. If you want, I can either commit it or leave a comment. I wasn't sure how you generate your files since there are no scrips in the project folder.

shyndman commented:
Could we get this modified to include only the docs? 

And thank you. I really appreciate you bringing symbol sprites to my attention.

jpc-ae commented:
Sure. Just have to rebase and clean some things up.
